@xen-orchestra/proxy/src/app/mixins/backups/index.js

```diff
@@ -57,7 +57,12 @@ export default class Backups {
     let run = ({ xapis, ...rest }) =>
       new Backup({
         ...rest,
-        app,
+
+        // don't change config during backup execution
+        config: app.config.get('backups'),
+
+        // pass getAdapter in order to mutualize the adapter resources usage
+        getAdapter: this.getAdapter.bind(this),
```

Comment:
I thought you wanted to share the adapters between multiple runs?

Reply:
Between multiple runs and all adapter's users.